### DISCUSSION TITLE
use different docker tag if .IsSnapshot == true

### DISCUSSION THREAD

**Discussion by cuotos** [goreleaser/goreleaser#2231] [Q&A] [answered]
Is it possible to use a different set of docker images tags if the build is a snapshot?

If .IsSnapshot is true I only want to build a single Docker image (v0.0.1-SNAPSHOT-xxxx) , if false then build a full set of `{{ .ProjectName }}:v{{ .Major }}.{{ .Minor }}"` images?

**Comment by caarlos0** [comment, marked as answer]
not possible right now I think 🤔

**Reply by ranjith-ka** [reply]
{{- if .IsSnapshot }}image:latest{{- else }}image:{{ .Version }}{{- end }}

May be !

**Reply by caarlos0** [reply]
yeah, that would work @ranjith-ka!
